onboarder/installation/index.rst

```diff
@@ -47,11 +47,35 @@ Build the UI
 
 Clear the Symfony cache and execute the following command to build the UI:
 
+If you have access to 'make' commands use :
 .. code-block:: bash
 
     rm -rf var/cache/*
     bin/console cache:warmup --env=prod
-    bin/console pim:installer:assets --clean --env=prod && yarn run webpack
+    make front
```

Comment:
Imo, it's better to specify the commands they have to launch instead as Pierrick have suggested in your previous PR

Reply:
I have added both 

make and bin/console commands